CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ for further information about branching and tagging conventions.
 - Updating of movements list and balances on Dashboard page after transfer
   performed
 - Updating of movements list on "Movements" page after transfer performed
+- Improvement of inserting Youtube video by a link
```

Comment:
1. Should be under `## [Unreleased]`. if no `## [Unreleased]` you can added at the top of the latest release
2. Rephrase "Allow inserting both of YouTube video link or ID"

Reply:
Rephrased and placed to correct place